### PR TITLE
Add blog post about Matter

### PR DESCRIPTION
**Pull request by balloob**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2023-02-08-state-of-matter-and-thread.markdown [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+As part of this effort, two years ago, we built ESP Web Tools. It revolutionized the DIY firmware world by allowing any ESP32/ESP8266 project to offer an easy web installer on their homepage, allowing users to get started directly from their browser. It has been adopted by all the great projects, including [Tasmota], [WLED], and [ESPresense]. Ever since the introduction of ESP Web Tools, we have wanted to expand this to more devices.
+
+Today we’re announcing [SL Web Tools][sl-web-tools]. It allows users to manage the firmware on Zigbee/Thread sticks using SiLabs chips directly from their browser, no terminals or compilation necessary. This allows any creator to offer easy firmware updates to their users and make experimental updates available to test bug fixes quickly.
+
+To see it in action, SL Web Tools has been added to [the Home Assistant SkyConnect documentation][skyconnect-docs].
+
+SL Web Tools is powered by our new open source Python package [`silabs-universal-flasher`][sl-flasher]. We run this in the browser using [Pyodide].
+
+# TODO: Video updating HA SkyConnect with SL Web Tools
```
Comment (balloob) [inline source/_posts/2023-02-08-state-of-matter-and-thread.markdown]: Pending todo

File: source/_posts/2023-02-08-state-of-matter-and-thread.markdown [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+## Building firmware for Silicon Labs chips
+
+It can be a pain to compile firmware for SiLabs chips. It’s hard to track different versions, your patch sets, and different chips. We’re fans of automation, so we have created an automated build system to create reproducible firmware builds for the Home Assistant Yellow and Home Assistant SkyConnect.
+
+We’re open sourcing this to make it easier for other creators to provide their users with up-to-date firmware. In combination with SL Web Tools, we offer a complete package that empowers other creators to provide their users with the latest firmware updates.
+
+[Find the repository on GitHub.][sl-builder]
+
+[matter]: /integrations/matter
+[rpidk]: https://raspberrypi.dk/en/?s=home+assistant+yellow&post_type=product
```
Comment (frenck) [inline source/_posts/2023-02-08-state-of-matter-and-thread.markdown:124]: This link is dead / forbidden

**Review by frenck — DISMISSED** [review]
Besides the pending todo and the dead link above; ✅

File: source/_posts/2023-02-08-state-of-matter-and-thread.markdown (hunk 2 above)
Comment (balloob) [reply source/_posts/2023-02-08-state-of-matter-and-thread.markdown:124]: was a spam filter. Is fixed now.

**Pull request merged**